velox/functions/lib/window/CMakeLists.txt

```diff
@@ -12,10 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_library(velox_functions_window NthValue.cpp)
+add_library(velox_functions_window INTERFACE)
```

Comment:
I don't get the reason why this should be removed. Is this because there is only 1 file?
There are some other spark window functions which has similar problems like nthvalue, for example, offset in ntile is also integer, rownumber's result type is integer instead of bigint. I think we need to move more windwos functions into this folder if we want to support them in spark.

Reply:
I don't think you need to define header-only library in CMake. Do you? What happens if you remove it?

Reply:
Removed, thanks~